test/trento/users_test.exs

```diff
@@ -114,6 +114,26 @@ defmodule Trento.UsersTest do
 
       assert analytics_enabled_at == nil
     end
+
+    test "update user profile sets analytics eula value with current time" do
+      user = insert(:user)
+
+      assert {:ok,
+              %User{
+                analytics_eula_accepted_at: analytics_eula_accepted_at
+              }} =
+               Users.update_user_profile(user, %{analytics_eula_accepted: true})
+
+      refute analytics_eula_accepted_at == nil
+
+      assert {:ok,
+              %User{
+                analytics_eula_accepted_at: analytics_eula_accepted_at
+              }} =
+               Users.update_user_profile(user, %{analytics_eula_accepted: false})
+
+      assert analytics_eula_accepted_at == nil
```

Comment:
we could maybe add a 3rd scenario where we don't set the `analytics_eula_accepted_at` and check that the value doesn't change:
```
assert {:ok,
    %User{
        analytics_eula_accepted_at: analytics_eula_accepted_at
      }} = Users.update_user_profile(user, %{})

assert analytics_eula_accepted_at == nil
```